dpppt-backend-sdk/dpppt-backend-sdk-ws/src/main/java/org/dpppt/backend/sdk/ws/config/WSBaseConfig.java

```diff
@@ -203,6 +213,40 @@ public ProtoSignature gaenSigner() {
     }
   }
 
+  @Bean
+  public InsertManager insertManager() {
```

Comment:
Nice! Maybe just write one or two lines of documentation here. Something that says "only accept keys that look valid". And maybe a ref to the modifiers if enabled.

Reply:
> Maybe just write one or two lines of documentation here

That's in fact the configuration part, the actual InsertManager is documented in its own class. See #235